source/jenkins/upgrade-java.html.md.erb

```diff
@@ -0,0 +1,76 @@
+---
+title: Upgrade Java for Jenkins
+last_reviewed_on: 2024-07-15
+review_in: 12 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+It is always a good idea to test things in SBOX before doing them in PROD, upgrading Java for Jenkins is no exception.
+Upgrade the agents first and then the built in controller node.
+
+## Upgrade Jenkins agents
+
+1. Create a PR in the [CNP Jenkins repo](https://github.com/hmcts/cnp-jenkins-docker/blob/master/jenkins/Dockerfile)  with the Java version incremented. You can find updated Jenkins Base image tag from the [Jenkins dockerhub repo](https://hub.docker.com/r/jenkins/jenkins/tags).
+2. Creating the PR should have automatically created a docker image with a pr- in it's name.
+3. Check the [container registry](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/8999dec3-0104-4a27-94ee-6588559729d1/resourceGroups/rpe-acr-prod-rg/providers/Microsoft.ContainerRegistry/registries/hmctspublic/repository) for the updated tag.
```

Comment:
Could you include that this goes into the `jenkins/jenkins` repository? There's a few jenkins ones in there